### PR TITLE
Enable Security Analysis (CodeQL) for Policy Compliance

### PR DESCRIPTION
**Pull request by MSNev**

### CODE REVIEW

File: projects/applicationinsights-angularplugin-js/src/lib/applicationinsights-angularplugin-js.component.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { Component } from '@angular/core';
 import {
-  IConfig, IPageViewTelemetry, IAppInsights, PropertiesPluginIdentifier, Util
+    IConfig, IPageViewTelemetry, IAppInsights, PropertiesPluginIdentifier, Util
```
Comment (MSNev) [inline projects/applicationinsights-angularplugin-js/src/lib/applicationinsights-angularplugin-js.component.ts:3]: Most of the changes here are whitespace only -- properly aligning the entire file to the same indent

File: projects/applicationinsights-angularplugin-js/src/lib/applicationinsights-angularplugin-js.component.ts [hunk 2]
```diff
@@ -14,96 +14,98 @@ import { ApplicationinsightsAngularpluginErrorService } from './applicationinsig
...
+                }
             }
-        }
-        if (identifier === PropertiesPluginIdentifier) {
-            this.propertiesPlugin = (ext as any) as properties.PropertiesPlugin;
-        }
-    });
-    if (extConfig.router) {
-        let isPageInitialLoad = true;
-        if (isPageInitialLoad) {
```
Comment (MSNev) [inline projects/applicationinsights-angularplugin-js/src/lib/applicationinsights-angularplugin-js.component.ts:64]: Removal of this if check as this was identified as an issue by codeQL

File: projects/applicationinsights-angularplugin-js/src/lib/applicationinsights-angularplugin-js.component.ts (hunk 1 above)
Comment (MSNev) [reply projects/applicationinsights-angularplugin-js/src/lib/applicationinsights-angularplugin-js.component.ts:3]: You can hide the whitespace changes using the gear drop down above (the one next to the "Jump to" to make life a LOT easier to read this 
image

**Review by kryalama — APPROVED** [review]
can we add steps on how to run codeQL locally in contributing guide?

**Comment by MSNev** [comment]
As CodeQL is a GitHub Action, I went looking for the documentation on running it locally and all I found was that they removed the documentation on how to do this (in a PR) -- so it looks like its not supported..

When you publish a PR it does run it and you can see the "Code scanning results / CodeQL" task is automatically added once the "CodeQL / Analyze" action completes.

**Pull request merged**